### PR TITLE
Judicial System: Implements email court date notifiactions to prison

### PR DESCRIPTION
**Pull request by gudjong**
# Court Date Notifications to Prison and Defender

https://app.asana.com/0/1197628843274291/1198563520384599
https://app.asana.com/0/1197628843274291/1198949971788358

Also adds the defender to the list of recipients of the signed ruling
https://app.asana.com/0/1197628843274291/1198949971788360

Also adds the defender to court date notifications to prosecutor and prison
https://app.asana.com/0/1197628843274291/1198987560584232

## What

☝️ 

## Why

Required feature

## Screenshots / Gifs

image

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against master before asking for a review

### CODE REVIEW

File: apps/judicial-system/backend/README.md [hunk 1]
```diff
@@ -22,9 +23,23 @@ a judge mobile number:
...
+```javascript
+emailOptions: {
+    useTestAccount: false,
+    options: {
+      region: 'eu-west-1',
+    },
+  },
+```
+
+Finally, to enamble prison email notifications provide a prison email address:
```
Comment (oddsson) [inline apps/judicial-system/backend/README.md:40]:
```suggestion
Finally, to enable prison email notifications provide a prison email address:
```

File: apps/judicial-system/backend/src/app/formatters/formatters.ts [hunk 2]
```diff
@@ -189,6 +190,32 @@ export function formatCourtDateEmailNotification(
...
+  isolation: boolean,
+): string {
+  const courtText = court.replace('dómur', 'dóms')
+  const courtDateText = formatDate(courtDate, 'PPPp')
+  const accusedGenderText =
+    accusedGender === CaseGender.OTHER
+      ? 'K'
+      : `Sakborningur er ${
+          accusedGender === CaseGender.MALE ? 'karl' : 'kona'
+        } og k`
```
Comment (oddsson) [inline apps/judicial-system/backend/src/app/formatters/formatters.ts:207]: Why doesn't this end with a space and  " og k" placed in line 216?

File: apps/judicial-system/web/src/routes/Judge/HearingArrangements/HearingArrangements.tsx [hunk 3]
```diff
@@ -368,7 +368,7 @@ export const HearingArrangements: React.FC = () => {
           {modalVisible && (
             <Modal
               title="Tilkynning um fyrirtökutíma hefur verið send"
-              text="Tilkynning hefur verið send á ákæranda með fyrirtökutíma."
+              text="Tilkynning hefur verið send á ákæranda með fyrirtökutíma. Fangelsi hefur líka fengið að vita af fyrirtökutímanum."
```
Comment (oddsson) [inline apps/judicial-system/web/src/routes/Judge/HearingArrangements/HearingArrangements.tsx:371]: Kemur þessi texti frá Önnu? Mér þætti eðlilegra að segja 

"Tilkynning hefur verið send á ákæranda og fangelsi með fyrirtökutíma."

**Review by oddsson — CHANGES_REQUESTED** [review]
👇

File: apps/judicial-system/backend/src/app/formatters/formatters.ts (hunk 2 above)
Comment (gudjong) [reply apps/judicial-system/backend/src/app/formatters/formatters.ts:207]: Rest of the sentence is there :)

"...rafist er gæsluvarðhalds til..."

This is covered in tests, which pass.

File: apps/judicial-system/web/src/routes/Judge/HearingArrangements/HearingArrangements.tsx (hunk 3 above)
Comment (gudjong) [reply apps/judicial-system/web/src/routes/Judge/HearingArrangements/HearingArrangements.tsx:371]: This text comes from Figma and will become more complicated when we add the defender as well.

"Tilkynning hafi verið send á ákæranda með fyritökutíma og verjanda hafi verjandi verið skráður. Fangelsi hefur líka fengið að vita af fyrirtökutímanum."

File: apps/judicial-system/backend/src/app/formatters/formatters.ts (hunk 2 above)
Comment (oddsson) [reply apps/judicial-system/backend/src/app/formatters/formatters.ts:207]: I saw the rest of the sentence in line 216, just thought it was weird 🤷‍♂️  I'd argue that you should change it for better readability...

File: apps/judicial-system/web/src/routes/Judge/HearingArrangements/HearingArrangements.tsx (hunk 3 above)
Comment (oddsson) [reply apps/judicial-system/web/src/routes/Judge/HearingArrangements/HearingArrangements.tsx:371]: Got it, thanks 🙏

File: apps/judicial-system/backend/src/app/formatters/formatters.ts (hunk 2 above)
Comment (gudjong) [reply apps/judicial-system/backend/src/app/formatters/formatters.ts:207]: You have a point there :) Will take a look.

File: scripts/schemas.js [hunk 4]
```diff
@@ -36,17 +36,23 @@ const main = async () => {
...
-        console.error(`Error while running generate-schemas: ${stderr}`)
-      }
+        .then((res) => {
+          console.log('-res', res)
+        })
+        .catch((e) => {
+          console.log('-e', e)
+        })
+
+      // console.log(stdout)
```
Comment (oddsson) [inline scripts/schemas.js:51]: Please remove this 🙏

**Review by oddsson — CHANGES_REQUESTED** [review]

File: scripts/schemas.js (hunk 4 above)
Comment (gudjong) [reply scripts/schemas.js:51]: Not my code. Jeremy is debugging broken builds.

**Pull request merged**